src/engine/input_name.asm

```diff
@@ -1245,32 +1337,47 @@ Func_1aefb:
 	ret nz
 	ld a, [wVisibleCursorTile]
 	bit 4, [hl]
-	jr z, Func_1afa1.asm_6fa4
+	jr z, DrawCursor_DeckNamingScreen
+;	fallthrough
 
-Func_1afa1:
+DrawInvisibleCursor_DeckNamingScreen:
```

Comment:
Same styling comments apply here

Reply:
If this isn't used outside the scope of this routine, why use it as a global label?